.github/scripts/check-release.sh

```diff
@@ -2,10 +2,18 @@
 
 # Checking if current tag matches the package version
 current_tag=$(echo $GITHUB_REF | cut -d '/' -f 3 | tr -d ' ',v)
-file_tag=$(grep '"version":' package.json | cut -d ':' -f 2- | tr -d ' ' | tr -d '"' | tr -d ',')
-if [ "$current_tag" != "$file_tag" ]; then
+
```

Comment:
I hesitated doing this! 

The issue with this is that the file would be changed but not committed with the changes to github. 
Another issue is that the `jest` tests are not run using rollup but using the typescript compiler. Thus we are not able to tests if the file has been updated correctly. Unless of course we consider the CI to be a good enough test.

Reply:
Also per example in the client of algolia, it would require to go by a shell script that writes over the file: https://github.com/algolia/algoliasearch-client-javascript/blob/master/scripts/build.js

Which is absolutely possible but we lose in understanding of the build process